provider/cldr/src/transform/decimal/mod.rs

```diff
@@ -13,7 +14,6 @@ use std::borrow::Cow;
 use std::convert::TryFrom;
 use tinystr::TinyStr8;
 
-mod cldr_serde;
 mod decimal_pattern;
 
 /// All keys that this module is able to produce.
```

Comment:
Remove the stray comment while you're `fmt`ing?

Reply:
This isn't a stray comment.  It documents the line below it.

Reply:
I'm referring to line 21 which I can't comment on.

Reply:
oh I see; that comment is there in order to make rustfmt format the key on its own line.